libs/openFrameworksCompiled/project/makefileCommon/config.linux.common.mk

```diff
@@ -131,7 +131,7 @@ ifeq ($(CXX),g++)
 			PLATFORM_CFLAGS = -Wall -std=c++0x -DHAS_TLS=0
 		else
 			ifeq ("$(GCC_MINOR_GTEQ_9)","1")
-				PLATFORM_CFLAGS = -Wall -std=c++14
+				PLATFORM_CFLAGS = -Wall -std=c++14 -DGCC_HAS_REGEX
```

Comment:
there's another line where this should be defined below this, where the gcc version comes from the compiler name